src/main/kotlin/io/github/rybalkinsd/kohttp/interceptors/LoggingInterceptor.kt

```diff
@@ -11,17 +12,28 @@ import okio.Buffer
  * Logs HTTP requests.
  *
  * @param log function to consume log message
+ * @param outputCurlCommand if true, output curl command of the request
  *
  * Sample Output: [2019-01-28T04:17:42.885Z] GET 200 - 1743ms https://postman-echo.com/get
  *
  * @since 0.8.0
  * @author gokul
  */
-class LoggingInterceptor(private val log: (String) -> Unit = ::println) : Interceptor {
+class LoggingInterceptor(
+        private val outputCurlCommand: Boolean = false,
+        private val log: (String) -> Unit = ::println
+) : Interceptor {
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
         val startTime = System.currentTimeMillis()
         return chain.proceed(request).also { response ->
+            if (outputCurlCommand) {
+                val command = request.buildCurlCommand()
+                log("╭--- cURL command -------------------------------")
+                log(command)
+                log("╰--- (copy and paste the above line to a terminal)")
```

Comment:
I think it is better to put it before ```kotlin chain.proceed(request)``` because we logging here only request

Reply:
I agree! Fixed in 054f8f0